app.rb

```diff
@@ -0,0 +1,101 @@
+require_relative 'book'
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'rental'
+require_relative 'person'
+
+class App
+  attr_accessor :books, :persons, :rentals
+
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+
+  # list all books
+  def list_books
+    puts 'List of books'
+    @books.each { |book| puts "title : #{book.title} by author : #{book.author}" }
+  end
```

Comment:
- [ Optional ] Kindly make sure that the the title and author are properly ordered in the case of your as shown below screen shoot it mention author in the place of title and it mention title in the place of the author Kindly Fix the issues. 
- ![test 1](https://github.com/supreeti/OPPS-Library-Management-System/assets/93598189/1a9267a8-95dd-4058-bfad-49839d53cb66)